docs/dev/netcore.md

```diff
@@ -2,33 +2,42 @@
 
 Building and packaging the agent, MUST be done with a specific version of the toolchain and .NET Core.  They are changing rapidly and our build process assumes this version.  We are looking into the env pulling that version.
 
-To build, we rely on .NET Core version 1.0.0.001598 
+To build, we rely on .NET Core SDK version 1.0.0-rc2-002416 
 
 Additional instructions are available [here](https://dotnet.github.io/getting-started/) Please use the links below to download the correct version of Windows/OSX installers.
 
 
 ## ![Win](../win_med.png) Windows
 
-Install [dotnet-win-x64.1.0.0.001598.exe](https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/1.0.0.001598/dotnet-win-x64.1.0.0.001598.exe)
+Install [dotnet-dev-win-x64.1.0.0-rc2-002416.exe](https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/1.0.0-rc2-002416/dotnet-dev-win-x64.1.0.0-rc2-002416.exe)
 
 > <sub><sup>TIP: Uninstall any existing version first (Programs and Features -> Microsoft Dotnet CLI for Windows).</sup></sub>  
 
 ## ![Apple](../apple_med.png) OSX 10.11  
 
-Install [dotnet-osx-x64.1.0.0.001598.pkg](https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/1.0.0.001598/dotnet-osx-x64.1.0.0.001598.pkg)
+Install [dotnet-dev-osx-x64.1.0.0-rc2-002416.pkg](https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/1.0.0-rc2-002416/dotnet-dev-osx-x64.1.0.0-rc2-002416.pkg)
+
+> <sub><sup>TIP: Uninstall any existing version first by deleting the folder that contains "dotnet" command.</sup></sub>  
 
 
 ## ![Linux](../linux_med.png) Ubuntu 14.04
 
-On Ubuntu, it must be installed using apt-get  
+On Ubuntu, it must be installed using dpkg  
 
 ```bash
-sudo sh -c 'echo "deb [arch=amd64] http://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list'  
+mkdir ~/dotnetpackages  
+
+wget -P ~/dotnetpackages https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/1.0.0-rc2-002416/dotnet-host-ubuntu-x64.1.0.0-rc2-002416.deb  
+
+wget -P ~/dotnetpackages https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/1.0.0-rc2-3002416/dotnet-sharedframework-ubuntu-x64.1.0.0-rc2-3002416.deb  
```

Comment:
It is weird and it may be an error on the CoreCLR team, but this is the version they published. I found by extracting the latest package and looking the folder name. I tried using initially the version without "3" prefix, but there is no such package.